bats/tests/helpers/profile.bash

```diff
@@ -256,14 +256,19 @@ count_setting_segments() {
 
 # Usage: profile_jq $expr
 #
-# Applies $expr against the profile and updates it in-places.
+# Applies $expr against the profile and update it in-place.
 profile_jq() {
     local expr=$1
     local filename
     filename=$(profile_location)
-    # Need to use a temp file to avoid truncating the file before it has been read.
-    jq "$expr" "$filename" | profile_cat "${filename}.tmp"
-    profile_sudo mv "${filename}.tmp" "$filename"
+    if [ -f "$filename" ]; then
+        # Need to use a temp file to avoid truncating the file before it has been read.
+        jq "$expr" "$filename" | profile_cat "${filename}.tmp"
+        profile_sudo mv "${filename}.tmp" "$filename"
+    else
+        # The profile doesn't exist yet; create a new file.
+        echo '{}' | jq "$expr" | profile_cat "$filename"
```

Comment:
I don't understand why you want to create the profile here. If all the `profile_add …` commands are expected to work without calling `create_profile` first, then similar logic would need to be added to `profile_plutil` as well.

So I would rather throw an error here than silently create a file that should already exist.

Reply:
Oh, are we always supposed to call `create_profile` before `profile_add`? I made this change because that _didn't_ happen with the `add_profile_bool "WSL.integrations` line in `vm.bash`, and it didn't look like the commented out path was doing that either.

I'm happy to just make things fall over if `create_profile` is required; that wasn't clear.

Reply:
> I made this change because that _didn't_ happen with the `add_profile_bool "WSL.integrations` line in `vm.bash`

Yes, we are supposed to always create the profile first. On Windows we get away with not calling it because it is a no-op, but we should always call it.

Reply:
One could argue that is `RD_USE_PROFILE` is true, the profile should have been created already by the caller, but I don't think this is always true. Not sure how this worked on non-Windows machines.

The whole `RD_USE_PROFILE` mechanism is an ugly hack that should go away. It only exists because we don't have a way yet to specify hash or list settings on `rdctl start` yet, so can't enable WSL integration during startup if we are also using a profile.